src/iop/lut3d.c

```diff
@@ -1048,7 +1144,7 @@ void gui_init(dt_iop_module_t *self)
   dt_bauhaus_combobox_add(g->interpolation, _("trilinear"));
   dt_bauhaus_combobox_add(g->interpolation, _("pyramid"));
   gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(g->interpolation) , TRUE, TRUE, 0);
-  gtk_widget_set_tooltip_text(g->interpolation, _("select the interpolation method"));
+  gtk_widget_set_tooltip_text(g->interpolation, _("select the interpolation method\n"));
```

Comment:
please no \n on tooltip. if space is needed this should be done in CSS for all tooltip anyway.

Reply:
done